northstar-tests/test-container/manifest.yaml

```diff
@@ -3,13 +3,15 @@ version: 0.0.1
 init: /test-container
 uid: 1000
 gid: 1000
-cgroups:
-  memory:
-    limit_in_bytes: 10000000
-    swappiness: 0
+# cgroups:
```

Comment:
Better add a TODO comment so we remember, why this was deactivated.

Reply:
Documented in #402